dirty_cat/test/test_minhash_encoder.py

```diff
@@ -40,6 +40,20 @@ def test_MinHashEncoder(n_sample=70, minmax_hash=False):
                 y_substring = encoder.transform(X_substring)
                 np.testing.assert_array_less(y - y_substring, 0.0001)
 
+def test_multiple_columns():
+    """ This test is intented to verify that fitting multiple columns
+        with the MinHashEncoder will not produce an error, but will 
+        encode the column independently """
+    from dirty_cat.datasets import fetch_employee_salaries
+    employee_salaries = fetch_employee_salaries()
+    X = employee_salaries.X
+    try:        
```

Comment:
I'm not sure that I understand the role of the try/except: if an error is raised, let it be raised. The test will fail, and that's good.